CodeStyleExercise/ColorsViewController.swift

```diff
@@ -8,55 +8,52 @@
 
 import UIKit
 
-class ColorsViewController : UIViewController, UICollectionViewDataSource, UITextFieldDelegate
+class ColorsViewController :
+    UIViewController,
+    UICollectionViewDataSource,
+    UITextFieldDelegate
 {
-
-    class constants {
-        static let DFLTNUMCOLORS:Int=12
-    }
-
+    
     @IBOutlet weak var collectionView:UICollectionView!
```

Comment:
There should be a space on the right side of the colon when defining type e.g. `collectionView: UICollectionView!`